pkgs/by-name/in/inori/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inori";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "eshrh";
+    repo = "inori";
+    tag = "v${version}";
+    hash = "sha256-D3umpem3Vj/T/UECe0qW7+qUpPgIJcRphIniURptgGE=";
+  };
+
+  cargoHash = "sha256-37rqKr+KtSuTI7pyah3nDWYnuusDyBdXjBHSrficPcE=";
+
+  meta = {
+    description = "Mpd client";
```

```suggestion
    description = "TUI Client for the Music Player Daemon (MPD)";
```
This isn't a very good description, even though it is used by the upstream repo. Lets take a bit from their README instead